src/SegmentedString.chpl

```diff
@@ -1449,6 +1457,13 @@ module SegmentedString {
     return interpretAsString(values, rng, borrow=true).isAlnum();
   }
 
+  /*
+    The SegFunction called by computeOnSegments for isalnum
```

Comment:
Looks good, I think you only forgot to switch this one to `isalpha`

Reply:
Thanks for pointing that out.  It was correct in my local copy.  Wondering if I pushed the wrong commit somehow?  Anyway, it should be fixed.  I pushed again and added an extra example in the unit test to differentiate better between isalpha and isalphanum.